index.html

```diff
@@ -42,15 +54,220 @@
             </p>
             <p class="connect">LET’S CONNECT</p>
             <ul class="social-container">
-              <li><i class="fab fa-twitter"></i></li>
-              <li><i class="fab fa-linkedin-in"></i></li>
               <li><i class="fab fa-github"></i></li>
-              <li><i class="fab fa-facebook"></i></li>
-              <li><i class="fab fa-instagram"></i></li>
+              <li><i class="fab fa-linkedin-in"></i></li>
+              <li><i class="fab fa-angellist"></i></li>
+              <li><i class="fab fa-twitter"></i></li>
+              <li><i class="fab fa-medium-m"></i></li>
             </ul>
           </section>
         </section>
       </section>
+
+      <section class="projects">
+        <section class="project">
+          <section class="project-image">
+            <img
+              src="./assets/images/project1.png"
+              alt="My first project snapshot"
+            />
+          </section>
+          <section class="project-name">Tonic</section>
+          <section class="project-year">
+            <span class="canopy">CANOPY</span>
+            <span class="dot"></span>
+            <span class="role">Back End Dev</span>
+            <span class="dot"></span>
+            <span class="role">2015</span>
+          </section>
+          <section class="project-description">
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required.
+          </section>
+          <section class="project-technologies">
+            <span>HTML</span>
+            <span>CSS</span>
+            <span>JavaScript</span>
+          </section>
```

Comment:
- [ ] The list of project technologies (e.g., "JavaScript", "HTML" and "CSS") are a list ul and each of them a li, please use ul and li instead of span

Reply:
I have changed this for all the "project-technology" classes.

Reply:
Well resolved